overrides/build-systems.json

```diff
@@ -7089,6 +7089,9 @@
   "gitlike-commands": [
     "poetry-core"
   ],
+  "gitlab-lint" : [
+    "poetry"
+  ],
```

Comment:
Formatting checks fails. This should be put before `"gitlike-commands"`.

Reply:
thank you! done and rebased onto default branch.